PhysicsTools/NanoAOD/python/muons_cff.py

```diff
@@ -119,6 +119,30 @@
     weightFile = "PhysicsTools/NanoAOD/data/mu_BDTG_2016.weights.xml",
 )
 
+run3_muon.toModify(
```

Comment:
in line 89, set `muonMVATTH` with the run3 parameters, and there do `run3_common).toModify(` with old paremters. which is just changing the fileinpath, the variable order and remove variables

Reply:
I see, I've updated the config to do this. I was not aware this `~run3` could be done. Let me know if it is what you expected.